ding/utils/data/dataset.py

```diff
@@ -69,6 +70,43 @@ def _load_d4rl(self, dataset: Dict[str, np.ndarray]) -> None:
             self._data.append(trans_data)
 
 
+@DATASET_REGISTRY.register('hdf5')
+class HDF5Dataset(Dataset):
+
+    def __init__(self, cfg: dict) -> None:
+        try:
+            import h5py
+        except ImportError:
+            logging.warning("not found h5py env, please install it")
```

Comment:
`please run "pip install h5py" to install it` can be better. And It caught ImportError, program should exit.